builder/builder.go

```diff
@@ -600,7 +600,7 @@ func (b *Builder) BuildUpdate(prefixflag string, minvflag int, formatflag string
 
 // BuildImage will now proceed to build the full image with the previously
 // validated configuration.
-func (b *Builder) BuildImage(format string, template string) {
+func (b *Builder) BuildImage(format string, template string) error {
```

Comment:
This makes line 620 (below) cause build error for me. Needs to be "return err". (Not sure why GitHub won't let me leave the comment on line 620 itself...)

Reply:
merge conflict messup :)